packages/angular-cli/commands/test.ts

```diff
@@ -5,6 +5,7 @@ import {CliConfig} from '../models/config';
 const NgCliTestCommand = TestCommand.extend({
   availableOptions: [
     { name: 'watch', type: Boolean, default: true, aliases: ['w'] },
+    { name: 'debug', type: Boolean, default: false, aliases: ['d'] },
```

Comment:
How is this option being passed into the karma config? I don't think it's being passed in yet.

My suggestion is this: in `karma.conf.js` add in the debug property to the `angularCli` options:
```
    angularCli: {
      config: './angular-cli.json',
      environment: 'dev',
      debug: false
    },
```

Then you'll need to also change `packages/angular-cli/tasks/test.ts` to set this to `true` in the `options` variable that's being passed into `karma.Server`.

Make sure it works though, this is the kind of thing I don't know we can really do CI tests without doing really weird stuff.